src/components/intro/intro.ts

```diff
@@ -4,15 +4,15 @@ import WebComponent, { createElementFromString } from '../../common/WebComponent
 
 class IntroComponent implements WebComponent {
   render(): Element {
-    return createElementFromString(`<main>
+    return createElementFromString(`<section>
         <div class="container">
```

Comment:
Last thing. Instead of wrapping `<div>` with `<section>` just use this `<section>` with `class="container"`. Same comes for tests.